docs/en/latest/plugins/error-log-logger.md

```diff
@@ -23,10 +23,16 @@ title: error-log-logger
 
 ## Summary
 
-- [**Name**](#name)
-- [**Attributes**](#attributes)
-- [**How To Enable And Disable**](#how-to-enable-and-disable)
-- [**How to set the TCP server address**](#how-to-set-the-tcp-server-address)
+- [Summary](#summary)
+- [Name](#name)
+- [Attributes](#attributes)
+  - [Attributes](#attributes-1)
+  - [Attributes(Batch-processor)](#attributesbatch-processor)
```

Comment:
This doesn't look good. How about adding a batch processor entry with the relevant hyperlink as an entry of the attributes table?